cni/Makefile

```diff
@@ -0,0 +1,6 @@
+TAG ?= $(shell git rev-parse --verify HEAD)
+
+build:
+	docker build -t kuma/install-cni:$(TAG) \
+		 -f build/Dockerfile \
+		 .
```

Comment:
I think you said build wasn't wired in but to make sure let's avoid having extra Makefiles outside of `mk/`